scripts/perf-test/run-on-ec2.sh

```diff
@@ -0,0 +1,256 @@
+#!/bin/bash
```

Comment:
:pick: This is a long bash file that looks to be mostly moved:
 - move in one commit, bug fixes and other changes in a separate one if possible
 - is bash the right language for something this long?